README.md

```diff
@@ -86,6 +87,56 @@ c.MyOAuthenticator.client_secret = 'your-client-secret'
 ```
 
 
+## Azure Setup
+
+* A sample is provided for you in `examples > azuread > sample_jupyter_config.py` 
+* Just add the code below to your `jupyterhub_config.py` file
+* Making sure to replace the values in `'{}'` with your APP, TENANT, DOMAIN, etc. values
+
+> ***Follow this [link to create an AAD APP](https://www.netiq.com/communities/cool-solutions/creating-application-client-id-client-secret-microsoft-azure-new-portal/)***
+
+> CLIENT_ID === Azure `Application ID` - found in AD --> app registrations --> app
+
+> TENANT_ID === Azure `Airectory ID` - found in AD --> properties
+
+
+
+
+Requires: **`PyJWT>=1.5.3`**
+
+```
+> pip3 install PyJWT
+```
+
+
+```
+import os
+os.environ["AAD_TENANT_ID"] = '{AAD-TENANT-ID}'
```

Comment:
rather than setting an environment variable with Python, maybe suggest setting the actual environment variable via `EXPORT` in the run.sh?

Reply:
done